internal/daysteps/daysteps.go

```diff
@@ -1,13 +1,60 @@
 package daysteps
 
+import (
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/Yandex-Practicum/tracker/internal/personaldata"
+	"github.com/Yandex-Practicum/tracker/internal/spentenergy"
+)
+
 type DaySteps struct {
-	// TODO: добавить поля
+	Steps    int
+	Duration time.Duration
+	personaldata.Personal
 }
 
 func (ds *DaySteps) Parse(datastring string) (err error) {
-	// TODO: реализовать функцию
+	parse := strings.Split(datastring, ",")
+
+	if len(parse) != 2 {
+		return fmt.Errorf("Ошибка парсинга DaySteps. Длина массива не 2")
+	}
+
+	steps, err := strconv.Atoi(parse[0])
+	if err != nil {
+		return fmt.Errorf("Ошибка парсинга. Шаги в число")
```

Comment:
1. Теряешь сообщение об ошибки
2. Ошибку лучше писать на английском
3. Ошибку лучше начинать с маленькой буквы
```suggestion
		return fmt.Errorf("conversion error: %w", err)
```

Reply:
Спасибо, поправил